src/main/java/de/eydamos/backpack/util/EnchUtils.java

```diff
@@ -0,0 +1,53 @@
+package de.eydamos.backpack.util;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.common.util.Constants.NBT;
+
+import de.eydamos.backpack.misc.ConfigurationBackpack;
+
+public class EnchUtils {
+
+    // -3 - disabled by config
+    // -1 - enchantment not found
+    private static final int SOUL_BOUND_ID = setSoulBoundID();
+
+    private EnchUtils() {}
+
+    private static int setSoulBoundID() {
+        if (!ConfigurationBackpack.ALLOW_SOULBOUND) return -3;
+
+        for (Enchantment ench : Enchantment.enchantmentsList)
+            if (ench != null && ench.getName().equals("enchantment.enderio.soulBound")) return ench.effectId;
+
+        return -1;
+    }
+
+    public static boolean isSoulBounded(ItemStack stack) {
+        NBTTagList stackEnch = stack.getEnchantmentTagList();
+        if (SOUL_BOUND_ID >= 0 && stackEnch != null) {
+            for (int i = 0; i < stackEnch.tagCount(); i++) {
+                int id = stackEnch.getCompoundTagAt(i).getInteger("id");
+                if (id == SOUL_BOUND_ID) return true;
+            }
+        }
+        return false;
```

Comment:
You can use `EnchantmentHelper` utility class to simplify code (and remove need in NBT manipulation).
```suggestion
        if (SOUL_BOUND_ID >= 0) {
            return EnchantmentHelper.getEnchantmentLevel(SOUL_BOUND_ID, stack) > 0;
        }
        return false;
```